raiden/transfer/channel.py

```diff
@@ -238,6 +240,7 @@ def is_channel_usable_for_new_transfer(
 
     channel_usable = (
         get_status(channel_state) == ChannelState.STATE_OPENED
+        and channel_state.settle_timeout >= channel_state.reveal_timeout * 2
```

Comment:
The signature of the function tells me there are multiple ways of defining the lock expiration, I couldn't verify that they are all correct.